sequence_processing_pipeline/Pipeline.py

```diff
@@ -133,6 +145,34 @@ class Pipeline:
 
     assay_types = [AMPLICON_ATYPE, METAGENOMIC_ATYPE, METATRANSCRIPTOMIC_ATYPE]
 
+    @staticmethod
+    def make_sif_fname(run_id, full_project_name):
```

Comment:
What's a sif?

Reply:
Okay. That's also the extension used by Singularity, which is used by at least Tell-Seq and anything else which originates as a Docker container. If "sif" is an acronym present within Qiita then keeping it makes sense. If it is an acronym introduced here, then reverting to the same terminology used in Qiita would helpful.

Reply:
Interestingly, the "sif files" (I can't stop saying that even though the "f" in "sif" stands for "files" ....) do NOT have the extension ".sif"--they are straight .tsv files.  In the Qiita user interface, I think these things are referred to as "sample info files"--which is what "sif" stands for, so it looks like there was an effort to keep terminology consistent  ...

Reply:
Yeah, at the time the wetlab was also using sif files (and Gail still does I believe) so it didn't seem like a good idea to change the extension on them.